app/feature/feature-claim-details/src/main/kotlin/com/hedvig/android/feature/claim/details/ui/ClaimDetailsDestination.kt

```diff
@@ -127,19 +155,123 @@ private fun ClaimDetailScreen(
+              showFileTypeSelectBottomSheet = false
+            },
+            modifier = modifier
+              .fillMaxWidth()
+              .padding(horizontal = 16.dp),
+            shape = MaterialTheme.shapes.squircleMedium,
+          ) {
+            Row(
+              horizontalArrangement = Arrangement.SpaceBetween,
+              modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 8.dp),
+            ) {
+              ProvideTextStyle(MaterialTheme.typography.bodyLarge) {
+                Text(text = stringResource(id = R.string.file_upload_photo_library))
+              }
+              Icon(imageVector = HedvigIcons.Pictures, contentDescription = "library icon")
+            }
+          }
+          Spacer(modifier = Modifier.height(4.dp))
+          TextButton(
+            onClick = {
+              photoCaptureState.launchTakePhotoRequest()
+              showFileTypeSelectBottomSheet = false
+            },
+            modifier = modifier
+              .fillMaxWidth()
+              .padding(horizontal = 16.dp),
+            shape = MaterialTheme.shapes.squircleMedium,
+          ) {
+            Row(
+              horizontalArrangement = Arrangement.SpaceBetween,
+              modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 8.dp),
+            ) {
+              ProvideTextStyle(MaterialTheme.typography.bodyLarge) {
+                Text(text = stringResource(id = R.string.file_upload_take_photo))
+              }
+              Icon(imageVector = HedvigIcons.Camera, contentDescription = "photo icon")
+            }
+          }
+          Spacer(modifier = Modifier.height(4.dp))
+          TextButton(
+            onClick = {
+              filePicker.launch("*/*")
+              showFileTypeSelectBottomSheet = false
+            },
+            modifier = modifier
+              .fillMaxWidth()
+              .padding(horizontal = 16.dp),
+            shape = MaterialTheme.shapes.squircleMedium,
+          ) {
+            Row(
+              horizontalArrangement = Arrangement.SpaceBetween,
+              modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 8.dp),
+            ) {
+              ProvideTextStyle(MaterialTheme.typography.bodyLarge) {
+                Text(text = stringResource(id = R.string.file_upload_choose_files))
+              }
+              Icon(imageVector = HedvigIcons.Document, contentDescription = "files icon")
+            }
+          }
+        }
+      },
+    )
+  }
+
+  LazyVerticalGrid(
+    columns = GridCells.Fixed(3),
     horizontalArrangement = Arrangement.spacedBy(8.dp),
-    verticalItemSpacing = 8.dp,
+    verticalArrangement = Arrangement.spacedBy(8.dp),
     modifier = modifier
       .windowInsetsPadding(WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal + WindowInsetsSides.Bottom))
       .padding(horizontal = 16.dp),
```

Comment:
LazyVerticalGrid should have a `contentPadding` parameter where we should pass these insets to. It may be something like 
```suggestion
    contentPadding = PaddingValues(horizontal = 16.dp) + WindowInsets.safeDrawing
      .only(WindowInsetsSides.Horizontal + WindowInsetsSides.Bottom)
```

We do something similar here https://github.com/HedvigInsurance/android/blob/85c852c395ff84b14e7010f0c3cba49d17138821/app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/HomeDestination.kt#L328-L331

I am mentioning this since otherwise the screen is cut-off at the bottom where the navigation items are.